manager.go

```diff
@@ -487,9 +487,8 @@ func (m *Manager) InitWithOptions(elf io.ReaderAt, options Options) error {
 	}
 
 	// set resource limit if requested
-	if m.options.RLimit != nil {
-		err := unix.Setrlimit(unix.RLIMIT_MEMLOCK, m.options.RLimit)
-		if err != nil {
+	if m.options.RemoveRlimit {
+		if err := rlimit.RemoveMemlock(); err != nil {
```

Comment:
Can we add a comment here mentioning that `RemoveMemlock` automatically detects whether there is cgroup accounting?